source/deploy/consider-mobile-vpn-options.rst

```diff
@@ -45,7 +45,7 @@ Connect via Corporate Proxy Server
 
 Review the following commonly-asked questions about connecting through a corporate proxy server:
 
-- `How do I receive mobile push notifications if my IT policy requires the use of a corporate proxy server? <https://docs.mattermost.com/deploy/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`__
-- `Deploy Mattermost with connection restricted post-proxy relay in DMZ or a trusted cloud environment <https://docs.mattermost.com/deploy/mobile-faq.html#deploy-mattermost-with-connection-restricted-post-proxy-relay-in-dmz-or-a-trusted-cloud-environment>`__
-- `Whitelist Mattermost push notification proxy to bypass your corporate proxy server <https://docs.mattermost.com/deploy/mobile-faq.html#whitelist-mattermost-push-notification-proxy-to-bypass-your-corporate-proxy-server>`__
-- `Run App Store versions of the Mattermost Mobile apps <https://docs.mattermost.com/deploy/mobile-faq.html#run-app-store-versions-of-the-mattermost-mobile-apps>`__
+- `How do I receive mobile push notifications if my IT policy requires the use of a corporate proxy server? <https://docs.mattermost.com/deploy/mobile-faq.html#how-do-i-receive-mobile-push-notifications-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`__
```

Comment:
Thanks for these changes. Could you elaborate on changing the URLs from the docs pages to the developer pages? The docs pages appear to be correct but if the developer content is more current than the docs content please let us know!

Reply:
The redirected link was coming like this. So changed it

Reply:
Apologies! I am referring to the links below this comment - they've been changed from docs links to links in the developer documentation. If the content in those docs is better or more accurate that'll be good to know so that we can plan to align them or consolidate them with the docs as opposed to having multiple sets of the same content. :)

Reply:
I've found the culprit, thanks for finding this @Sayanta66! Very good catch. The content has actually been moved to the docs set, so we'll retain all the links as they currently are. :)

Reply:
If you could revert all but the first one, that'd be awesome. :)

Reply:
Thanks a lot. Doing the needful

Reply:
Resolved